manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -542,23 +546,26 @@ func (r *FybrikApplicationReconciler) checkGovernanceActions(configEvaluatorInpu
 	var err error
 	switch configEvaluatorInput.Request.Usage {
 	case taxonomy.WriteFlow:
-		if !req.Context.Requirements.FlowParams.IsNewDataSet {
+		if !req.Context.Requirements.FlowParams.IsNewDataSet ||
+			req.Context.Requirements.FlowParams.ResourceMetadata != nil {
```

Comment:
This code is reached by either writing a new asset or updating an existing asset. In both cases it is problematic to use the supplied metadata.

Reply:
we will revisit this part. ResourceMetadata from flow requirements should be taken into account only when `isNewDataSet` is true